frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipeline-task-runs-display.feature

```diff
@@ -46,3 +46,22 @@ Feature: Display task runs page
              Then user is redirected to Task Run Details tab
               And user can see "Details", "Log", "YAML" and "Events" tab
               And user can see Status, Message and Log snippet in "Details" tab
+
+
+        @regression, @manual
+        Scenario Outline: Task Runs Details page with Workspaces
+            Given user is at pipeline details page with pipeline runs with all kind of Workspaces
```

Comment:
Curious -- wouldn't it be easier to just start on a Pipeline Run or Task Run details page? Any reason why you chose Pipeline as the starting ground?

I take it the
> with pipeline runs with all kind of Workspaces

Is the way we express the "Examples" table below?

Reply:
Updated